doc/source/examples/04_applied/03_ahmed_body_fluent.mystnb

```diff
@@ -0,0 +1,360 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+# Applied: Develop an External Aerodynamic Simulation Model for Fluent Analysis
+
+The Ahmed body is a simplified car model used to study airflow around vehicles. The wake (turbulent flow behind the body)
+ depends on the slant angle:
+
+ - Less than 12°: The airflow stays attached to the slant, creating low drag and a mostly two-dimensional flow.
+ - 12° to 30°: The flow becomes three-dimensional with strong c-pillar vortices, peaking at 30°. This increases
+  drag due to low-pressure areas on the rear surfaces.
+ - More than 30°: The flow fully separates from the slant, reducing drag and weakening the c-pillar vortices.
+
+In this example we are going to create an Ahmed body with a slant angle of 20 degrees, The following steps are involved:
+ 1. Launch Pyansys geometry and define the default units
+ 2. Creation of sketches for ahmed body, enclosure and BOI.
+ 3. Generating Solid bodies from the sketches.
+ 4. Performing boolean operations for region extraction.
+ 5. Grouping faces and defining named selection.
+ 6. Export model as a CAD file.
+ 7. Close session.
+
+### Define function for sorting planar face pairs along any axis
+
+This function is used to sort the planar faces along any of the coordinate axis. This is used primarily for sorting the faces to define
+the named selections.
+
+```{code-cell} ipython3
+def face_identifier(faces, axis):
+    """
+    Sorts a pair of planar faces based on their positions along the specified coordinate axis.
```

Comment:
```suggestion
    Sort a pair of planar faces based on their positions along the specified coordinate axis.
```
The description for a function should start with a verb that doesn't have a "s"--so "Sort" rather than "Sorts" in this case.  For descriptions of arguments, we omit a leading article (such as The, A, or An) so that the first one is an important one. Subsequent sentences that provide more information should use articles and be complete sentences.